ApplyToBecomeInternal/ApplyToBecome.Data.Tests/Services/UserRepositoryTests.cs

```diff
@@ -16,107 +15,18 @@ namespace ApplyToBecome.Data.Tests.Services
 	public class UserRepositoryTests
 	{
 		private readonly Fixture _fixture = new Fixture();
-
+	
 		[Theory, AutoMoqData]
-		public async Task SearchUsers_MatchesGivenName_ReturnsUsers([Frozen] Mock<IGraphUserService> graphUserService,
```

Comment:
Why remove this test? Is it because we'll not be searching via endpoint and filtering our end now?

Reply:
I've changed the implementation to return all users. The autocomplete control now carries out the filter.
In general I'd say its better to have your search logic on the server rather than client side, though that's not really how this control was designed to work.